README.md

```diff
@@ -118,37 +49,26 @@ Use the command `ape networks list` to see the networks available:
 
 ```sh
 ethereum  (default)
+├── holesky
+│   ├── chainstack
+│   └── node  (default)
+├── local  (default)
+│   ├── node
+│   └── test  (default)
 ├── mainnet
-│   ├── geth  (default)
-│   └── chainstack
-├── ropsten
-│   ├── geth  (default)
-│   └── chainstack
-├── kovan
-│   └── geth  (default)
-├── rinkeby
-│   ├── geth  (default)
-│   └── chainstack
-├── goerli
-│   ├── geth  (default)
-│   └── chainstack
-└── local  (default)
-    ├── geth
-    └── test  (default)
+│   ├── chainstack
+│   └── node  (default)
+└── sepolia
+    ├── chainstack
+    └── node  (default)
 ```
 
 Use the `--network` command to access the console using your node; for example:
 
 ```bash
-ape console --network ethereum:goerli:chainstack
+ape console --network ethereum:sepolia:chainstack
 ```
 
 Check the Ape docs to see [how to select a network](https://docs.apeworx.io/ape/stable/userguides/networks.html).
 
-Now you are ready to use Ape to develop and test your smart contract, checkout the [Ape Academy](https://academy.apeworx.io/) for tutorials.
-
-## Development
```

Comment:
I think the chainstack plugin is still considered to be in development since the recent releases have only been in alpha so this section should still be relevant.

Reply:
True! I added back the notice

[commit](https://github.com/ApeWorX/ape-chainstack/pull/10/commits/0e70c27543241e15d20f4494473bbad9f9c2f816)